ui/core/sim_concurrent.ts

```diff
@@ -115,6 +112,12 @@ function runSims(requests: RaidSimRequest[], totalIterations: number, wp: Worker
 		for (let i = 0; i < requests.length; i++) {
 			wp.raidSimAsync(requests[i], pm => progressHandler(i, pm));
 		}
+
+		signals.abort.onTrigger(() => {
+			for (const req of requests) {
+				wp.abortById(req.id);
```

Comment:
Same here, do they need to be awaited?

Reply:
No, this will cause the workers to abort and the progress handler above will pick up the first final error result coming in due to that and pass that on as the error result.